src/main/kotlin/blackjack/view/OutputView.kt

```diff
@@ -1,18 +1,18 @@
 package blackjack.view
 
 import blackjack.model.Dealer
+import blackjack.model.Playable
 import blackjack.model.Player
-import blackjack.utils.Constants
 
 object OutputView {
     fun displayInitialState(
         players: List<Player>,
         dealer: Dealer,
     ) {
-        val names = listOf(dealer.name) + players.map { it.name.trim() }
+        val names = listOf("Dealer") + players.map { it.name.trim() }
         val sentence = names.joinToString(", ")
         println("\nDealing two cards to $sentence.")
-        println("${dealer.name}: ${dealer.hand.cards[0]}")
+        println("Dealer: ${dealer.hand.cards[0]}")
         players.forEach { displayCurrentHand(it) }
     }
```

Comment:
Finally! Even if the way the dealer is represented changes to an image file, you'll only need to check the view layer 💯